### DISCUSSION TITLE
Traffic Monitor couldn't call the API of API-Builder to extract traffic from Elasticsearch

### DISCUSSION THREAD

**Discussion by lyr3051** [Axway-API-Management-Plus/apigateway-openlogging-elk#195] [Q&A] [answered]
Hello , this issue is related to  #179.

My API-Builder run as container on my VM elk, and APIM run on VM apim.
And the version of release package is 4.2.0 .

After I create a new instances.yml to generate certificate like below:
<img width="239" alt="image" src="https://user-images.githubusercontent.com/43023893/159685613-3ef07d37-c9dd-4f9d-94c0-82f293ad6d42.png">

My the url of API-Builder is https://elk:8443, and my API_BUILDER_SSL_KEY and API_BUILDER_SSL_CERT are elk.crt and elk.key.
I also add the following new environment variable: env.API_BUILDER_URL=https://elk:8443 in <apigateway-install-dir>/apigateway/conf/envSettings.props.
And conf/fed is imported the provided Policy-Fragement(Ver. 7.7.0) from release package 4.2.0.

However when I try to apply to search traffic on Traffic Monitor, it fails to extract traffic from Elasticsearch, and return the below error message: (more trace log please find the below attached)
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/43023893/159687750-6866b851-8fca-47f5-9771-ac2c91be9212.png">
[nodemanager_trace.txt](https://github.com/Axway-API-Management-Plus/apigateway-openlogging-elk/files/8332520/nodemanager_trace.txt)

Could you help me to resolve this issue?

**Comment by cwiechmann** [comment, marked as answer]
Have you imported the generated CA into the ANM-Configuration already?

**Reply by lyr3051** [reply]
Hi @cwiechmann 
Thanks for your help.
I imported the generated CA from conf/certificate release package 4.2.0, the Traffic Monitor extract traffic from Elasticsearch Successfully.
